CONTRIBUTING.md

```diff
@@ -3,6 +3,14 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
+## How to build
+Fulfill [gtk-rs dependencies](http://gtk-rs.org/docs/requirements.html) for cairo crate
```

Comment:
readme.md states it's just cargo run, while without this dependency I'm getting compilation errors.
Remove the statement from readme.md?

Reply:
If you can't build on windows without specifying the target, could you open an issue? I'm not sure if that's a known thing or not.

In any case, I'm a bit confused: windows should be using d2d, so you shouldn't need the cairo dependency I don't think?

And related, the instructions added here are, at the very least, windows specific, so if we want to add this info it would be worth having separate `### windows` and `### macos` sections I think!

Reply:
The mention of `cargo run` in the readme is more referring to what is involved in getting a client application to run.

Reply:
I'm using GNU/Linux (Ubuntu 18.4 dependencies, Mint 19) and cannot compile because my platform have no implementation. And for me the only way to find it out was to read the code and see, that it is not a dependency problem, there is not implementation at all and I have to specify supported platform explicitly.

Reply:
Ah, yes okay that makes sense. :)

I think the solution to this problem is to clearly state in the readme what platforms are supported. We could also just deny compilation based on the current platform, e.g. something like,

```rust
#[cfg(not(any(target_os = "macos", target_os = "windows")))]
compile_error!("druid does not current support your platform.");
```

which would at least give a clear and immediate diagnostic? I'm not sure this is necessary though, it feels a bit cute.

Reply:
I think this small check make sense
In general, supporting GNU/Linux is something I'm planning to work on, but I would like to play around with it before declaring to maintain a platform. I'm new in Rust yet.

Regarding this docs, I will note that gtk-rs needed only for MacOs targeted builds, links at the bottom and general readme update regarding two platforms, right?

Reply:
updated readme